unrasterize/classes.py

```diff
@@ -126,7 +126,7 @@ def _sort_pixels(band):
         )
 
     @staticmethod
-    def _reassign_pixel_values(band, pixels, raw_pixel_values=[]):
+    def _reassign_pixel_values(threshold, band, pixels, raw_pixel_values=[]):
         """Adjust values of selected pixels so that their sum is preserved.
 
         Parameters
```

Comment:
Please also update the docstring with a description of the new function argument.